chai-as-promised/chai-as-promised.d.ts

```diff
@@ -32,4 +32,4 @@ declare module chai {
         isRejected(promise: any, expected: any, message?: string): void;
         isRejected(promise: any, match: RegExp, message?: string): void;
     }
-}
\ No newline at end of file
+}
```

Comment:
Oh, you were talking about the closing curly brace. Yeah. It wasn't removed. GitHub is smoking. See https://github.com/jedmao/DefinitelyTyped/blob/fix-chai-libs/chai-as-promised/chai-as-promised.d.ts